drf_jsonapi/mixins.py

```diff
@@ -154,12 +158,12 @@ def create(self, request):
 
         # Check for relationships and process them
         if 'relationships' in request.data['data']:
-            resource = self.process_to_one_relationships(request.data['data']['relationships'], resource)
+            resource = self.process_to_one_relationships(request.data['data']['relationships'], resource, request)
```

Comment:
do we need to pass `request.data` as an arument separately from `request`, or can we have the called function extract the data it needs?

Reply:
I can see arguments for either side. I left it this way to make it backwards compatible and to make testing easier. The `request` is not STRICTLY required but the request data and the resource are. Passing the request here allows the implementation to use it to make user-specific decisions.